### ISSUE TITLE
Synced computers do not disappear from the list.

### ISSUE THREAD

**Issue by SZYKowny** [pluginsGLPI/ocsinventoryng#284]
- The version of the plugin: **1.7.3**
- The version of your GLPI: **9.5.4**
- The steps to reproduce your issue.

Hi,
i upgraded glpi 9.4.6 -> 9.5.4 and plugin 1.6.2 -> 1.7.3

After that update syncs works fine but some computers still remain on the non sync list.
Only computers with DEBUG = 65536 are problems.

I cant see any php or sql errors.

**Comment by SZYKowny** [comment]
When switch off software sync, its work fine.
I think problem was before, i upgraded ocsinventory-ng to version 8 and use glpi 9.4.6, plugin 1.6.2. In that configuration syncs works only when software switch off.
How to find out what's broken?

**Comment by tsmr** [comment]
No error into php-errors.log or sql-errors.log ?

**Comment by SZYKowny** [comment]
Yes, no errors. All works good. Computers on the sync list with DEBUG = 65536 not disapear from sync list.
If they have DEBUG <> 65536 they are syncs good and back to DEBUG = 65536 and stay on list.
If switch off software to sync i have clear list after sync.

**Comment by tsmr** [comment]
And in Debug mode you have errors on synchronize computer ?

**Comment by SZYKowny** [comment]
No errors in debug mode.
In debug mode i see 1 more column, name DEBUG. My up notes says about that column.
